.circleci/config.yml

```diff
@@ -0,0 +1,70 @@
+version: 2
+jobs:
+  build:
+    parallelism: 4
+    docker:
+      - image: golang:1.8
+    environment:
+      GOVERSION: "1.8.3"
+      GOPATH:  "/root/.go_workspace"
+      WORKDIR: "/root/.go_workspace/src/k8s.io/helm"
+      PROJECT_NAME: "kubernetes-helm"
+    steps:
+      - run:
+          name: update PATH
+          command: echo 'export PATH=$GOPATH/bin:$PATH' >> $BASH_ENV
+      - run:
+          name: remove old go files
+          command: rm -rf /usr/local/go && rm -rf "$GOPATH"
+      - run:
+          name: install go
+          command: wget "https://storage.googleapis.com/golang/go${GOVERSION}.linux-amd64.tar.gz" -O "${HOME}/go${GOVERSION}.tar.gz" && tar -C /usr/local -xzf "${HOME}/go${GOVERSION}.tar.gz"
+      - checkout
+      - run:
+          name: move repository to the canonical import path
+          command: mkdir -p "$(dirname ${WORKDIR})" && mv "${HOME}/project" "${WORKDIR}"
+      - run:
+          name: install dependencies
+          command: cd "${WORKDIR}" && make bootstrap
+      - run:
+          name: go env
+          command: go env
+      - run:
+          name: test
+          command: cd "${WORKDIR}" && ./scripts/ci.sh
+  deployment:
+    docker:
+      - image: golang:1.8
+    environment:
+      GOVERSION: "1.8.3"
+      GOPATH:  "/root/.go_workspace"
+      WORKDIR: "/root/.go_workspace/src/k8s.io/helm"
+      PROJECT_NAME: "kubernetes-helm"
+    filters:
+      tags:
+        only: /.*/
+      branches:
+        only: master
+    steps:
+      - run:
+          name: update PATH
+          command: echo 'export PATH=$GOPATH/bin:$PATH' >> $BASH_ENV
+      - run:
+          name: remove old go files
+          command: rm -rf /usr/local/go && rm -rf "$GOPATH"
+      - run:
+          name: install go
+          command: wget "https://storage.googleapis.com/golang/go${GOVERSION}.linux-amd64.tar.gz" -O "${HOME}/go${GOVERSION}.tar.gz" && tar -C /usr/local -xzf "${HOME}/go${GOVERSION}.tar.gz"
```

Comment:
The installing go steps can be removed because it is running in the `golang:1.8` image